### DISCUSSION TITLE
Installing PyRosetta on Windows 11 (No WSL)

### DISCUSSION THREAD

**Discussion by Kalsol04** [RosettaCommons/rosetta#80] [PyRosetta] [answered]
Hello... is there any way to install PyRosetta on native Windows 11 without using Windows Subsystem for Linux?

**Comment by lyskov** [comment, marked as answer]
Unfortunately not at this time, - right now we only have support for Mac and Linux systems

**Comment by roccomoretti** [comment]
Marking this as "closed", as I hope your question has been answered. If that assessment is incorrect, feel free to re-open or open up a new discussion with more details.
